### PR TITLE
2023-01-13 Ernest Kissiedu

### PR DESCRIPTION
**Pull request by allenwyma**

### CODE REVIEW

**Comment by ernestkissiedu** [comment]
Hi Allen, 

I just listened to the audio, and it's great. You can hear the difference from when I moved from the room to the booth. I guess next time. I should use the booth. 

Thanks again for this opportunity.

**Review by ernestkissiedu — APPROVED** [review]
Hi Allen, 

This looks good on my end. Happy to go ahead and publish. Thanks again for the opportunity.

File: _episodes/interviews/2023-01-13-ernest-kissiedu.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "Rust Nation with Ernest Kissiedu"
+date: 2023-01-13T17:54:00Z
+file: https://audio.rustacean-station.org/file/rustacean-station/2023-01-13-ernest-kissiedu.mp3
+duration: "52:02"
+length: "49964929"
+#reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
+---
+Allen Wyma talks with [Ernest Kissiedu](https://github.com/ernestkissiedu), Organizer of [Rust Nation](https://www.rustnationuk.com/), a Rust-focused conference geared towards Rust beginners and Rust curious.
```
Comment (jonhoo) [inline _episodes/interviews/2023-01-13-ernest-kissiedu.md:9]:
```suggestion
Allen Wyma talks with [Ernest Kissiedu](https://github.com/ernestkissiedu), organizer of [Rust Nation](https://www.rustnationuk.com/), a Rust-focused conference geared towards Rust beginners and the Rust-curious.
```

File: _episodes/interviews/2023-01-13-ernest-kissiedu.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+- [@0:00] - Introduction
+- [@3:21] - Rust Nation's purpose and goal
+- [@5:14] - Ernest's background and being the community lead of Rust London Community
+- [@13:29] - Rust community all around the world
```
Comment (jonhoo) [inline _episodes/interviews/2023-01-13-ernest-kissiedu.md:24]:
```suggestion
- [@13:29] - Rust communities all around the world
```

File: _episodes/interviews/2023-01-13-ernest-kissiedu.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+- [@16:14] - Possible guests and speakers on the Rust conference.
```
Comment (jonhoo) [inline _episodes/interviews/2023-01-13-ernest-kissiedu.md:25]:
```suggestion
- [@16:14] - Possible guests and speakers at the Rust conference.
```

File: _episodes/interviews/2023-01-13-ernest-kissiedu.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+- [@22:39] - Why you should join the Rust conference regardless of your expertise in Rust
```
Comment (jonhoo) [inline _episodes/interviews/2023-01-13-ernest-kissiedu.md:26]:
```suggestion
- [@22:39] - Why you should attend the Rust conference regardless of how experienced you are with Rust
```

**Review by jonhoo — APPROVED** [review]

**Pull request merged**